components/Navbar.tsx

```
@@ -1,15 +1,22 @@
 'use client';
 
+import { useEffect, useState } from 'react';
 import { NavigationMenu, NavigationMenuItem, NavigationMenuList } from './ui/navigation-menu';
```

ESLint

[error] Delete `useEffect,·useState·}·from·'react';⏎import·{·`

(prettier/prettier)